src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/NotifyLiPClaimantHwFOutcomeHandlerTest.java

```diff
@@ -131,12 +135,76 @@ void shouldNotifyApplicant_HwfOutcome_NoRemission_Hearing() {
             // Then
             verify(notificationService, times(1)).sendMail(
                 EMAIL,
-                EMAIL_TEMPLATE_HWF,
+                EMAIL_TEMPLATE_NO_REMISSION,
                 getNotificationDataMapNoRemissionHearing(),
                 REFERENCE_NUMBER
             );
         }
 
+        @Test
+        void shouldNotifyApplicant_HwfOutcome_RefNumberUpdated_ClaimIssued() {
+            // Given
+            HelpWithFeesDetails hwfeeDetails = HelpWithFeesDetails.builder()
+                .hwfCaseEvent(UPDATE_HELP_WITH_FEE_NUMBER)
+                .build();
+            CaseData caseData = CaseDataBuilder.builder().atStateClaimSubmitted().build().toBuilder()
+                .applicant1(PartyBuilder.builder().individual().build().toBuilder()
+                                .partyEmail(EMAIL)
+                                .build())
+                .respondent1Represented(YesOrNo.NO)
+                .specRespondent1Represented(YesOrNo.NO)
+                .applicant1Represented(YesOrNo.NO)
+                .caseDataLiP(CaseDataLiP.builder().helpWithFees(HelpWithFees.builder().helpWithFeesReferenceNumber(HWF_REFERENCE).build()).build())
+                .claimIssuedHwfDetails(hwfeeDetails)
+                .hwfFeeType(FeeType.CLAIMISSUED)
+                .build();
+
+            CallbackParams params = CallbackParamsBuilder.builder().of(ABOUT_TO_SUBMIT, caseData).build();
+
+            // When
+            handler.handle(params);
+
+            // Then
+            verify(notificationService, times(1)).sendMail(
+                EMAIL,
+                EMAIL_TEMPLATE_UPDATE_REF_NUMBER,
+                getNotificationDataMapUpdateRefNumberClaimIssued(),
+                REFERENCE_NUMBER
+            );
+        }
+
+        @Test
+        void shouldNotifyApplicant_HwfOutcome_RefNumberUpdated_Hearing() {
+            // Given
+            HelpWithFeesDetails hwfeeDetails = HelpWithFeesDetails.builder()
+                .hwfCaseEvent(UPDATE_HELP_WITH_FEE_NUMBER)
+                .build();
+            CaseData caseData = CaseDataBuilder.builder().atStateClaimSubmitted().build().toBuilder()
+                .applicant1(PartyBuilder.builder().individual().build().toBuilder()
+                                .partyEmail(EMAIL)
+                                .build())
```

Comment:
Reuse Case Data from as created in recent changes. Please check Pablo's test cases.

Reply:
Done